osimpipeline/study.py

```diff
@@ -209,6 +270,13 @@ def add_task_cycles(self, cls, *args, **kwargs):
                 args = orig_args + (setup_tasks[i],)
 
             task = cycle.add_task(cls, *args, **kwargs)
+
+            if (("Inverse Kinematics" in task.doc) or 
+                ("Inverse Dynamics" in task.doc)):
+                raise Exception("TrialTask creation for individual cycles not "
+                    " current supported for the Inverse Kinematics and "
```

Comment:
I'd prefer the `type(task)` syntax, but checking the doc attribute covers all the setup, tool, and post tasks at once.